scripts/generate-cipherinfo.py

```diff
@@ -0,0 +1,337 @@
+#!/usr/bin/env python
```

Comment:
That's the version which only adds what the configuration in mumble.ini allows isn't it? Have to say that while the previous version was more complex it seemed to create a decent lookup table that should cover most customizations users might attempt. As this is displayed in the client I would prefer it being flexible in what might come from the server.